test/versioned/hapi/hapi-v20/package.json

```diff
@@ -1,16 +1,16 @@
 {
-  "name": "hapi-v17-tests",
+  "name": "hapi-v20-tests",
   "version": "0.0.0",
   "private": true,
   "tests": [
     {
       "engines": {
-        "node": ">=12 < 16"
+        "node": ">=14"
       },
       "dependencies": {
         "ejs": "2.5.5",
-        "hapi": "^17.0.0",
-        "vision": "5.1"
+        "@hapi/hapi": ">=20.1.2",
+        "vision": "latest"
```

Comment:
Noting this has changed from a single version to "latest." That is fine, it didn't used to be possible but is now.

I did take a quick check and `vision`, without a namespace, is no-longer maintained. I'm not sure we even have support for `@hapi/vision`. I'm also not sure if that instrumentation is needed anymore for Hapi. We may want to capture a story to dig into that.

Reply:
@michaelgoin I'll add a story for that.